redfish-core/lib/log_services.hpp

```diff
@@ -4339,6 +4351,41 @@ inline void requestRoutesSystemDumpEntryCollection(App& app)
             std::ref(app)));
 }
 
+inline void requestRoutesSystemDumpEntryDownload(App& app)
+{
+    // Hardware dump
+    BMCWEB_ROUTE(
+        app,
+        "/redfish/v1/Systems/system/LogServices/Dump/Entries/0<str>/attachment/")
```

Comment:
Its because we will need to have two paths: System & Resource dumps and all other bmc stored host dumps.

I tested with A<str>, B<str> and others as just /<str>/ in the URI. But I remember getting errors for URL/route not found when I used it this way. Hence I defined separate routes for these bmc stored host dumps (IBM specific)

Reply:
but these are not valid redfish urls right?  If that is the case why don't your try something like
/redfish/v1/Systems/system/LogServices/Dump/Entries/A/<str>/attachment/
/redfish/v1/Systems/system/LogServices/Dump/Entries/B/<str>/attachment/
/redfish/v1/Systems/system/LogServices/Dump/Entries/<str>/attachment/
better to replace A & B with better readable name